client/sites/settings/administration/tools/delete-site/index.jsx

```diff
@@ -254,6 +254,7 @@ class DeleteSite extends Component {
 
 		return (
 			<Panel className="settings-administration__delete-site">
+				<HeaderCakeBack icon="chevron-left" onClick={ this._goBack } />
```

Comment:
I initially tried keep the old layout for nav unification as-is (via feature flag); however, due to the nature of the multi-pages it's quite complicated. I don't think it's worth it; especially since these pages are isolated (full-page layout) and are one-off features 🙈 I hope it's OK